### PR TITLE
UserGuide format tweaks

### PR DESCRIPTION
**Pull request by realqijun**
# Resolves
* #298 (Sorry Simon),
* #315

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -152,30 +152,30 @@ The **GUI** is divided into five main sections:
...
+| **Find**     | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find Apple Inc`                                                                                                                        |
+| **View**     | `view INDEX`                                                                                                                                                                 |
+| **Apply**    | `apply INDEX n/NAME d/DESCRIPTION [as/APPLICATION_STATUS]`<br> e.g., `apply 1 n/Software Engineering Intern d/Uses React`                                                        |
+| **Update**   | `update c/COMPANY_INDEX app/APPLICATION_INDEX as/APPLICATION_STATUS`<br> e.g.,`update c/1 app/1 as/OA`                                                                       |
+| **Withdraw** | `withdraw c/COMPANY_INDEX app/APPLICATION_INDEX`<br> e.g., `withdraw c/3 app/1`                                                                                              |
+| **Reopen**   | `reopen INDEX`                                                                                                                                                                   |
```
Comment (blanklogic) [inline docs/UserGuide.md:168]: Think according to #298 of reorganising commands to "Company", "Application", "Misc.", Reopen belongs under Company instead.

**Review by blanklogic — CHANGES_REQUESTED** [review]
Great addition of pictures, some nits regarding where `reopen` should be placed. 

Also regarding the organisation of User Guide I have some questions regarding #298 . Should we make the segregation more obvious with header tags for Company commands, Application commands and General commands for the entire User Guide?

**Comment by uniqly** [comment]
@blanklogic @realqijun Regarding the organisation with header, I think that is good. I've seen other groups do that also.

**Review by blanklogic — APPROVED** [review]
Thanks for making the requested changes. LGTM!

**Review by uniqly — APPROVED** [review]
LGTM!

**Pull request merged**